charts/k8ssandra/templates/reaper/reaper.yaml

```diff
@@ -32,4 +32,22 @@ spec:
   {{- if .Values.reaper.tolerations }}
   tolerations: {{ toYaml .Values.reaper.tolerations | nindent 4 }}
   {{- end }}
+  schemaInitContainerConfig:
+    {{- if (and .Values.reaper.schemaInitContainerConfig .Values.reaper.schemaInitContainerConfig.securityContext) }}
```

Comment:
Could you have used the `ternary` function here like you did elsewhere?

Reply:
Yes, we can,  it just included an extra bit of checking for the existence of the config entry itself and looked a bit messy to stuff all of that into a one-liner.